src/FTPObject.jl

```diff
@@ -268,6 +270,7 @@ Set the transfer mode to binary.
 """ ->
 function binary(ftp::FTP)
 
+    ftp.ctxt.options.binary_mode = true
```

Comment:
Function names ending with an `!` indicate that the function is modifying the arguments. Mainly it's just a [nice convention](http://docs.julialang.org/en/release-0.4/stdlib/base/#introduction) to dictate that the function will be modifying a parameter.